src/components/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -34,6 +34,37 @@ export interface SegmentedControlOption {
   disabled?: boolean
 }
 
+// functions used in prop validators
+const getValues = (items: SegmentedControlOption[]) => {
+  const vals:string[] = []
+  items.forEach((item: SegmentedControlOption) => vals.push(item.value + ''))
+
+  return vals
+}
+
+const itemValuesAreUnique = (items: SegmentedControlOption[]): boolean => {
+  const vals = getValues(items)
+  const uniqueValues = new Set(vals)
+
+  return vals.length === uniqueValues.size
+}
+
+const validateItems = (items: SegmentedControlOption[] | string[]): boolean => {
+  const nItems = normalizeItems(items)
+
+  return itemValuesAreUnique(nItems)
+}
```

Comment:
You need to move this function definition below the `normalizeItems` function so that you are not calling it before it is defined.